WORKSPACE.bazel

```diff
@@ -125,6 +129,28 @@ nlohmann_json()
 prometheus_cpp_deps()
 ### Dependencies for orc8r/gateway/c/common
 
+### CPP_REDIS DEPENDENCIES ###
+cpp_redis()
+### CPP_REDIS DEPENDENCIES ###
+
 ### DEPENDENCY FOR TESTING
 cpp_testing_deps()
 ### DEPENDENCY FOR TESTING
+
+### Folly dependencies ###
+git_repository(
+    name = "rules_foreign_cc",
+    commit = "a7f1c9e6046a98b6006e183f0112f4815bf2ebc0",
+    remote = "https://github.com/bazelbuild/rules_foreign_cc",
+)
+
+load("@rules_foreign_cc//foreign_cc:repositories.bzl", "rules_foreign_cc_dependencies")
+
+# Don't use preinstalled tools to ensure builds are as hermetic as possible
+rules_foreign_cc_dependencies(register_preinstalled_tools = False)
```

Comment:
Yeah I think I must have copied and pasted this from an example - will look into what it does and either comment or remove :D

Reply:
https://github.com/bazelbuild/rules_foreign_cc/blob/main/docs/README.md#rules_foreign_cc_dependencies

i think setting this flag to False is correct. AFAICT, it tells bazel not to look for pre-installed cmake/ninja on the path. `register_built_tools` (default True) should take over and download/build cmake/ninja inside the bazel sandbox for use.

do we want to set cmake_version or ninja_version here? otherwise, rules_foreign_cc has some defaults (3.21.0/1.10.2)

Reply:
Yes, I think setting the versions explicitly will be better.